DS-Session-6-Assignment/src/question1/GraphImplementation.java

```diff
@@ -0,0 +1,222 @@
+package question1;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+import java.util.Stack;
+
+/**
+ * This class implements the methods of the Graph interface
+ * @author Jugal
+ * Dated 08/09/2019
+ */
+public class GraphImplementation implements Graph {
+
+	private int[][] graph;
+	private int noOfVertices;
+
+	/**
+	 * Constructor to initialize the data members of the class 
+	 * @param noOfVertices is the number of vertices of a graph
+	 */
+	GraphImplementation(int noOfVertices) {
+		if(noOfVertices <= 0)
+		{
+			System.out.println("Enter Some amount of vertices.");
+		}
+		else
+		{
+			this.noOfVertices = noOfVertices;
+			graph = new int[this.noOfVertices][this.noOfVertices];
+		}
+	}
+	
+	/**
+	 * This method adds an edge to the graph
+	 * @param source is the source vertex
+	 * @param destination is the destination vertex
+	 * @param weight is the weight of the edge
+	 * @return {boolean}
+	 */
+	boolean addEdgeToTheGraph(int source, int destination, int weight)
```

Comment:
You should not limit graph nodes to an integer only. They might be in the form of A, B, C .... etc. So Implement the program so we can give any type of nodes.